glean-core/tests/ping.rs

```diff
@@ -101,3 +102,89 @@ fn empty_pings_with_flag_are_sent() {
     assert_eq!(false, ping2.submit(&glean, None).unwrap());
     assert_eq!(1, get_queued_pings(glean.get_data_path()).unwrap().len());
 }
+
+#[test]
+fn test_pings_submitted_metric() {
+    let (mut glean, _temp) = new_glean(None);
+
+    let pings_submitted = LabeledMetric::new(
```

Comment:
Can you add a comment saying that we reconstruct it here to check its values instead of reaching within the library internals?